packages/element-library/src/media/scalePanel.js

```diff
@@ -24,11 +24,10 @@ import { Slider } from '@googleforcreators/design-system';
 import { BG_MIN_SCALE, BG_MAX_SCALE } from '@googleforcreators/animation';
 import { InOverlay } from '@googleforcreators/moveable';
 
-/**
- * Internal dependencies
- */
-import { Z_INDEX_CANVAS } from '../../constants';
-
+const Z_INDEX_CANVAS = {
+  MOVABLE: 10,
+  FLOAT_PANEL: 11,
+};
```

Comment:
This feels a little slippery - z index is specific to the editor, maybe I'm missing something about how things are getting bucketed.  We've got work coming up next sprint on addressed the z-index issues pertaining to the floating menus and I'm a little apprehensive about this move until I understand the thought process better about how the boundaries here are getting drawn. Apologies if I missed a comment or thread somewhere.

Reply:
The only reason why it was moved here is that the only place `Z_INDEX_CANVAS` is being used in this file. When it is required in more than one place, we can think of its proper home.

Reply:
I second Brittany here that this feels out of place, even if it's used only in this file. There are some other Z_INDEX constants out there in the editor, e.g.
```
export const Z_INDEX = {
  NAV: 2,
  EDIT: 3,
};
```

Maybe there's a place where these constants should be combined then in the editor? These z-indexes are editor-specific. Otherwise we could also set the values to 1 and 2 here -- independently it should work ok, it just wouldn't work within the editor anymore. If we'll keep them as it is, would be great to add a comment about why these are 10 and 11.
Thoughts?

Reply:
> Would be happy to move, but I cannot think of any suitable home for `Z_INDEX_CANVAS`.

Since all these z-indexes are in theory editor-specific and determine how different layers are positioned on top of each other in the editor, then ideally, this should somehow come from the implementation (the editor).

That said, perhaps we could verify first that it actually does influence how things are displayed in the editor -- since it's currently used in one place only, maybe it's not related to all the other z-indexes in the editor. In this case just renaming the constant to something else to avoid confusion could be ok.

But if it indeed influences how the layers are displayed in the editor, then probably a follow-up issue to investigate it / figure out a solution would be necessary.

Thoughts?

Reply:
Oh okay, I think now I understand the concern a little bit. We can keep it in `story-editor` and maybe pass it down to `ScalePanel` via props.

Reply:
Does this look good? https://github.com/GoogleForCreators/web-stories-wp/pull/10452/commits/1c2577106d1f4fd84b6fe7b0842c42c5337196a7

Reply:
Looking at it again, I'm seeing that the `zIndex.MOVABLE` is not even used at all anywhere? 🤔 
In this case, we could just pass one z-index directly. WDYT?

```
function ScalePanel({
  setProperties,
  width,
  height,
  x,
  y,
  scale,
  zIndex,
  ...rest
}) {
  return (
    <InOverlay zIndex={zIndex} pointerEvents="initial">
  ...
```

Reply:
Since `MOVABLE` is currently not being used we can remove it from the object but still pass the same object so that more keys can be passed in the future.